components/card/card.html

```diff
@@ -0,0 +1,430 @@
+                        style="width: 100%; height: 361px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                      </iframe>
+
+                    </code>
+                </div>
+            
+            </section>                                  
+            <hr class="line-horz">                               
+            <section>
+
+                <h2 class="headline2">Card with Text OVerLay</h2>
```

Comment:
Mistake: "OVerLay" may be it should "Overlay". everything else is good.

Reply:
yess